src/actions/actions.ts

```diff
@@ -1652,6 +1652,114 @@ class MoveLineBegin extends BaseMovement {
 }
 
 @RegisterAction
+class MoveScreenLineBegin extends BaseMovement {
+  modes = [ModeName.Normal, ModeName.Visual, ModeName.VisualLine];
+  keys = ["g", "0"];
+  viewPosition = "wrappedLineStart";
```

Comment:
Will do. I'm naming it as `viewPosition` as what we do in Code, still remember the `viewColumn` naming I used before :)